sphinx_gallery/notebook.py

```diff
@@ -191,5 +195,6 @@ def python_to_jupyter_cli(args=None, namespace=None):
     for src_file in args.python_src_file:
         file_conf, blocks = split_code_and_text_blocks(src_file)
         print('Converting {0}'.format(src_file))
-        example_nb = jupyter_notebook(blocks)
+        gallery_conf = copy.deepcopy(gen_gallery.DEFAULT_GALLERY_CONF)
```

Comment:
Sorry if I missed it, why doesn't this actually use the currently active gallery conf and just use the default one instead? It's worth adding a comment to justify because it's strange

Reply:
Maybe I'm not understanding the CLI, but it seemed to me that there is no "gallery configuration" when using the CLI, so I was just copying the default one. Maybe put another way, what do you mean by "use the currently active gallery conf" in the context of the CLI?

Reply:
What do you mean by CLI? You mean calling `sphinx_build` or something else?

Reply:
This is just an utility function. It is not really related to the Sphinx-Gallery build, but a way to circumvent it just for one task. What it does is to directly parse the python script and output a jupyter notebook. I think @choldgraf is right in doing the copy of the default config here. Also this function has no access to the configuration at build time, and there is no guarantee this would even be executed within a project with Sphinx-Gallery for us to search for that config file. Finally I could be wrong, but I guess not many people use this feature, although the reverse (from notebook to script) has been requested. Thus we can leave this default for now.

Reply:
Ahh okay, makes sense.

Reply:
I know the TLA @choldgraf :) I just had no idea SG had a CLI aside from using it with `sphinx-build` calls!